js/storage.js

```diff
@@ -1,3 +1,97 @@
+// TODO: Move these helper fns elsewhere
```

Comment:
I'm not sure where would be a better place to move these functions so that they are accessible both in this file and `client-teambuilder.js`. It seems sloppy to make them global like this (also, ESLint is complaining and saying it can't find them from `client-teambuilder.js` causing CI to fail, though I can run these changes locally with no errors).